pkg/services/star/star_model.go

```diff
@@ -0,0 +1,36 @@
+package star
+
+import "errors"
+
+var ErrCommandValidationFailed = errors.New("command missing required fields")
+
+type Star struct {
+	Id          int64
+	UserId      int64
+	DashboardId int64
```

Comment:
Also, it might be worth mentioning as TODO that in the commands below DashboardIDs should become UIDs instead at some point?